tests/ui/literals.rs

```diff
@@ -7,10 +7,10 @@
 
 fn main() {
     let ok1 = 0xABCD;
-    let ok3 = 0xab_cd;
-    let ok4 = 0xab_cd_i32;
-    let ok5 = 0xAB_CD_u32;
-    let ok5 = 0xAB_CD_isize;
+    let ok3 = 0xabcd;
+    let ok4 = 0xabcd_i32;
+    let ok5 = 0xABCD_u32;
+    let ok5 = 0xABCD_isize;
```

Comment:
These changes to the test file should be unnecessary now?

Reply:
Yep!